docs/reference.md

```diff
@@ -4,3 +4,14 @@
 - [Coding Guidelines](../docs/reference/coding-guidelines.md)
 - [Testing Overview](../docs/reference/testing-overview.md)
 - [Frequently Asked Questions](../docs/reference/faq.md)
+
+## Logo
+<img width="200px" src="https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/final-g-wapuu-black.svg?sanitize=true" alt="Gutenberg Logo" />
```

Comment:
The `width` here doesn't need `px`. Most modern browsers will totally ignore text in this field (try setting the width to `200in` and see it still renders at 200px). Width/height attributes are implicitly `px` according to the spec.